app/components/subject-viewer.cjsx

```diff
@@ -71,6 +71,10 @@ module.exports = React.createClass
       clearTimeout @signInAttentionTimeout
       @setState loading: true
 
+  componentDidUpdate: (prevProps) ->
```

Comment:
Because `getInitialFrame` acts on `this.props` so we wait until after the value updates.